lib/pages/AthletePage.dart

```diff
@@ -757,12 +760,14 @@ class _AthletePageState extends State<AthletePage> {
                                                                 false,
                                                                 false))),
                                                   ]),
-                                                  Text(bookValue,
+                                                  Visibility(
```

Comment:
Why are we using a visibility widget if we are not going to toggle the visibility at all?